13-tool-usage-cycle-tracking/README.md

```diff
@@ -1,15 +1,13 @@
-# Remote Power Control
-
-Remote Control power, and monitor line and load power quality and report power failures.
+# Tool Cycle-Usage Tracking
 
+Monitor digital inputs, monitor power inputs, monitor vibration, optionally alert when vibration is out of range when the tool is active.
 ## Solution Overview
 
-This solution uses two Dr. Wattson monitoring boards to monitor AC line power and AC load power, with the load power switched on or off using a remote controllable relay. If you do not wish to monitor AC line power going into the relay this solution can be deployed with just one Dr. Wattson instance to monitor load power and report power failures.
-
-The solution comprises hardware, firmware, Notehub environment variables and jsonata scripts to provide power quality monitoring, power control and power quality alerts.
+This solution uses a [Dr. Wattson energy monitoring board] to monitor a tool's power supply and power usage. The tool, or a component connected to the tool, provides a 3.3-5v digital signal indicating when the tool is active. You can configure the solution with upper and lower thresholds for voltage, current and power and receive alerts when the monitor detects values out of range. The solution also senses vibration using the Notecard's built-in accelerometer, and this too can be used to generate alerts for over vibration.
```

Comment:
> You can configure the solution with upper and lower thresholds for voltage, current and power and receive alerts when the monitor detects values out of range.

You can configure the solution with upper and lower thresholds for voltage, current and power, and receive alerts when the monitor detects values out of range.